Resources/views/Form/fields.html.twig

```diff
@@ -4,7 +4,7 @@
 {%- endblock %}
 
 {% block vich_file_widget %}
-    {% spaceless %}
+    {% apply spaceless %}
```

Comment:
We can't just replace `spaceless` with `apply spaceless`, since this bundle is not requiring any specific version of Twig. So, if some users use an old version of Twig, their code would break.

I see two possible solutions: 
* add a `conflict` section in our composer.json (hard)
* use the whitespace "minus" operator instead (easy)

Reply:
from this

```
{% block vich_file_widget %}
    {% apply spaceless %}
        <div class="vich-file">
            {{ form_widget(form.file) }}
            {% if form.delete is defined %}
                {{ form_row(form.delete) }}
            {% endif %}

            {% if download_uri %}
                <a href="{{ download_uri }}">{{ translation_domain is same as(false) ? download_label : download_label|trans({}, translation_domain) }}</a>
            {% endif %}
        </div>
    {% endapply %}
{% endblock %}
```

to this ?

```
{%- block vich_file_widget -%}
    <div class="vich-file">
        {{ form_widget(form.file) }}
        {% if form.delete is defined %}
            {{ form_row(form.delete) }}
        {% endif %}

        {% if download_uri %}
            <a href="{{ download_uri }}">{{ translation_domain is same as(false) ? download_label : download_label|trans({}, translation_domain) }}</a>
        {% endif %}
    </div>
{%- endblock -%}
```

Reply:
That's not enough, you should use `{{-` and `-}}` too

Reply:
Moreover: also other `{%` should be changed to  `{%-`

Reply:
alright, will update the PR